cli/src/main/java/de/medavis/lct/cli/CreateManifest.java

```diff
@@ -44,12 +44,14 @@ class CreateManifest implements Callable<Void> {
     private File outputFile;
     @Option(names = {"--template", "-t"})
     private String template;
+    @Option(names = {"--ignoreUnavailableUrl", "-iuu"}, defaultValue = "true")
```

Comment:
Unterscheidet das sich nicht von der Beschreibung in der README? In der Readme steht default "false".

Reply:
I'll take the liberty to answer in English ;-)

You are right, CLI and Jenkins should be consistent in their default values. I have changed it. Thanks!